Zend/zend_language_scanner.h

```diff
@@ -61,19 +61,6 @@ typedef struct _zend_lex_state {
 	zend_arena *ast_arena;
 } zend_lex_state;
 
-typedef struct _zend_heredoc_label {
```

Comment:
I don't think it makes sense to make these private, as they can be accessed via the public struct via heredoc_label_stack. Similar issue for zend_nest_location.